packages/core/src/lib/base_protocol.ts

```diff
@@ -45,73 +41,4 @@ export class BaseProtocol implements IBaseProtocolCore {
   protected async getStream(peer: Peer): Promise<Stream> {
     return this.streamManager.getStream(peer);
   }
-
-  /**
-   * Returns known peers from the address book (`libp2p.peerStore`) that support
-   * the class protocol. Waku may or may not be currently connected to these
-   * peers.
-   */
-  public async allPeers(): Promise<Peer[]> {
-    return getPeersForProtocol(this.components.peerStore, [this.multicodec]);
-  }
-
-  public async connectedPeers(): Promise<Peer[]> {
```

Comment:
Do we still have access to this utility function from elsewhere?
Basically getting the connected peers for a particular protocol